Mathlib/Analysis/SpecialFunctions/PolarCoord.lean

```diff
@@ -89,20 +89,24 @@ def polarCoord : PartialHomeomorph (ℝ × ℝ) (ℝ × ℝ) where
     · exact (Complex.continuousAt_arg hz).continuousWithinAt
     · exact Complex.equivRealProdCLM.symm.continuous.continuousOn
 
+/-- The derivative of `polarCoord.symm`, see `hasFDerivAt_polarCoord_symm`. -/
+def fDerivPolarCoordSymm (p : ℝ × ℝ) : ℝ × ℝ →L[ℝ] ℝ × ℝ :=
```

Comment:
No, `fderivPolarCoordSymm`. When the first letter is not capitalized, immediately following letters that could be capitalized are not either. Yes, the naming conventions are complicated.